crates/typst/src/math/attach.rs

```diff
@@ -240,10 +264,10 @@ macro_rules! measure {
 
 /// Layout the attachments.
 fn layout_attachments(
-    ctx: &mut MathContext,
```

Comment:
Because the call site builds `[tl, t, tr, bl, b, br]` in place, and they need the `ctx` variable as a mutable, since `MathContext::layout_into_fragment()` takes a `mut &self`.
Therefore, I need to pass `ctx` here later than the `[tl, t, tr, bl, b, br]`.

Reply:
Could you move the array construction into a `let` binding?

Reply:
Done.

Reply:
And then move the ctx back up. I know it's very minor, but it's just the canonical style.

Reply:
Just moved, done.